shell/browser/native_window_mac.mm

```diff
@@ -198,14 +199,6 @@ void ReorderChildWindowAbove(NSWindow* child_window, NSWindow* other_window) {
   bool hiddenInMissionControl = false;
   options.Get(options::kHiddenInMissionControl, &hiddenInMissionControl);
 
-  bool useStandardWindow = true;
-  // eventually deprecate separate "standardWindow" option in favor of
-  // standard / textured window types
-  options.Get(options::kStandardWindow, &useStandardWindow);
-  if (windowType == "textured") {
-    useStandardWindow = false;
-  }
```

Comment:
I'm :+1: on just removing this option without emitting a deprecation warning. It was removed from the docs in **2015** when d427ae10 migrated from that option to `type`.